### ISSUE TITLE
edit parcel error

### ISSUE THREAD

**Issue by holysanta** [kartoza/parcel_plugin#61]
I get this error when i try to edit a parcel:

```
2018-04-12T06:30:33	1	Traceback (most recent call last):
			  File "C:/Users/me/.qgis2/python/plugins\parcel_plugin\plugin.py", line 364, in manage_parcels
			    ParcelManager(self.iface, self.database, self.required_layers)
			  File "C:/Users/me/.qgis2/python/plugins\parcel_plugin\plugin.py", line 588, in __init__
			    self.run()
			  File "C:/Users/me/.qgis2/python/plugins\parcel_plugin\plugin.py", line 665, in run
			    data)
			  File "C:/Users/me/.qgis2/python/plugins\parcel_plugin\PyQt4Dialogs.py", line 1016, in __init__
			    self.populate_form(data)
			  File "C:/Users/me/.qgis2/python/plugins\parcel_plugin\PyQt4Dialogs.py", line 1034, in populate_form
			    self.SQL_PARCELS["SELECT"], (feat_id,))[0][0]
			  File "C:/Users/me/.qgis2/python/plugins\parcel_plugin\database.py", line 108, in query
			    'Error raised: %s.' % (str(e),))
			Exception: Backend database query failed!
			Error raised: can't adapt type 'function'.
```

**Comment by NyakudyaA** [comment]
@holysanta Are you using the plugin from the QGIS plugin repository.  I will try it and see if I can replicate. Can you send me a database dump so that I can try to replicate on my machine? In the meantime, I will try with the sample survey you sent me earlier.

**Comment by holysanta** [comment]
Yes..plugin is from repository. Database dump sent to you on friday.

**Comment by NyakudyaA** [comment]
@holysanta I will release an update today. So far we have fixed the #62
